app/login/page.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from '@playwright/test';
+
+const correctCredentials = {
+  email: 'testemail@email.com',
+  password: 'test1234',
+};
+const incorrectCredentials = {
+  email: 'wrongemail@email.com',
+  password: 'wrongpassword',
+};
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/login');
+});
+
+test.describe('Tests login page', () => {
+  test('should successfully login', async ({ page }) => {
+    await page.getByTestId('email').fill(correctCredentials.email);
+    await page.getByTestId('password').fill(correctCredentials.password);
```

Comment:
I would go for user-facing attributes like `getByRole` or `getByPlaceholder` since we are testing user interaction.

Reply:
Funnily enough I *did* but Shashi kind of said otherwise.

Reply:
Do you know the reason?